toonz/sources/toonz/startuppopup.cpp

```diff
@@ -369,21 +387,17 @@ void StartupPopup::showEvent(QShowEvent *) {
       m_recentNamesLabels[i] = new StartupLabel(justName, this, i);
       m_recentNamesLabels[i]->setToolTip(
           name.remove(0, name.indexOf(" ") + 1));  // remove "#. " prefix
-      m_recentSceneLay->addWidget(m_recentNamesLabels[i], i, Qt::AlignTop);
+      m_recentSceneLay->addWidget(m_recentNamesLabels[i], 0, Qt::AlignTop);
       i++;
     }
-    m_recentSceneLay->addStretch(100);
   }
-
   bool ret = true;
   for (int i = 0; i < m_recentNamesLabels.count() && i < 7; i++) {
```

Comment:
Sorry for the late notice.
For now, 8th, 9th and 10th item in the list cannot be opened on clicked since the connection is limited to 1-7th items here.